docs/data/material/components/grid2/grid2.md

```diff
@@ -0,0 +1,240 @@
+---
+product: material-ui
+title: React Grid component
+githubLabel: 'component: Grid'
+materialDesign: https://material.io/design/layout/understanding-layout.html
+---
+
+# Grid version 2
+
+<p class="description">The responsive layout grid adapts to screen size and orientation, ensuring consistency across layouts.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+The `Grid` component works well for a layout with known columns. The columns can be configured in multple breakpoints which you have to specify the column span of each child.
+
+## How it works
+
+The grid system is implemented with the `Grid` component:
+
+- It uses [CSS's Flexible Box module](https://www.w3.org/TR/css-flexbox-1/) for high flexibility.
+- The grid is always a flex item. Use the `container` prop to add flex container to it.
+- Item widths are set in percentages, so they're always fluid and sized relative to their parent element.
+- There are five default grid breakpoints: xs, sm, md, lg, and xl. If you need custom breakpoints, check out [custom breakpoints grid](#custom-breakpoints).
+- Integer values can be given to each breakpoint, indicating how many of the 12 available columns are occupied by the component when the viewport width satisfies the [breakpoint constraints](/material-ui/customization/breakpoints/#default-breakpoints).
+- It uses negative margin and padding technique to create [gap-like](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) between children.
+- It **does not** have the concept of rows. Meaning, you can't make the children span to multiple rows. If you need to do that, we recommend to use [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) instead.
+- It **does not** offer auto-placement children feature. It will try to fit the children one by one and if there is not enough space, the rest of the children will start on the next line and so on. If you need the auto-placement feature, we recommend to use [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout) instead.
```

Comment:
```suggestion
- It does _not_ automatically place children. It will try to fit the children one by one, and if there is not enough space, the rest of the children will start on the next line, and so on. If you need auto-placement, we recommend using [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout) instead.
```